### PR TITLE
docs: Add example for kube-apiserver entity policy

### PR DESCRIPTION
**Pull request by joestringer**

### CODE REVIEW

**Review by christarazi — APPROVED** [review]

File: examples/policies/l3/entities/apiserver.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[{
+    "labels": [{"key": "name", "value": "dev-to-host"}],
```
Comment (ghost) [inline examples/policies/l3/entities/apiserver.json]:
```suggestion
    "labels": [{"key": "name", "value": "dev-to-kube-apiserver"}],
```

File: examples/policies/l3/entities/apiserver.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+apiVersion: "cilium.io/v2"
+kind: CiliumNetworkPolicy
+metadata:
+  name: "dev-to-host"
```
Comment (ghost) [inline examples/policies/l3/entities/apiserver.yaml]:
```suggestion
  name: "dev-to-kube-apiserver"
```

**Review by ghost — COMMENTED** [review]
Left a suggestion to change the name, dev-to-host is a bit confusing

File: examples/policies/l3/entities/apiserver.json (hunk 1 above)
Comment (joestringer) [reply examples/policies/l3/entities/apiserver.json]: Oh yeah :facepalm: . Bit too much copypasta going on there. Fixed :+1:

**Review by ghost — APPROVED** [review]

**Pull request merged**